test/DropdownSpec.js

```diff
@@ -257,4 +257,26 @@ describe('<Dropdown>', () => {
     const wrapper = mount(<Dropdown />);
     expect(wrapper.find('div').length).to.equal(1);
   });
+
+  it('Should also accept a custom component', () => {
+    const customComponent = React.forwardRef(
+      (
+        {
+          // eslint-disable-next-line no-unused-vars
+          show,
```

Comment:
Would be neat if we could test the attributes for correctness. But I don't see there any easy way to do this via Jest, because we would have to pass them here via DOM attributes to the actual DOM component. Then we could check for the correctness via expect etc. and compare the props.
Any ideas?